### DISCUSSION TITLE
WebSocket Request Going to ws-mt1.pusher.com instead of MyServer.com

### DISCUSSION THREAD

**Discussion by RizwanNaasir** [filamentphp/filament#5314] [Help]
Hello Guys, I hope you are doing okay!

I have setup Laravel WebSocket Community Package running, when Configure the echo in Filament Config The Request on Browser console is going to pusher's hosts instead of mine.
Can Point in the right direction that where I can change the host.
Thank you so much.

**Comment by bikash-s-bhandari** [comment]
@RizwanNaasir  i was facing the same issue, it was due to incorrect configuration 

```
'echo' => [
            'broadcaster' => 'pusher',
            'key' => env('MIX_PUSHER_APP_KEY'),
            'cluster' => env('MIX_PUSHER_APP_CLUSTER'),
            'wsHost' => env('MIX_PUSHER_HOST'),
            'wsPort' => env('MIX_PUSHER_PORT'),
            'wssPort' => env('MIX_PUSHER_PORT'),
            'authEndpoint' => '/api/broadcasting/auth',
            'disableStats' => true,
            'encrypted' => true,
        ],
```
